lib/screens/v2/widget_instance/elevator_status.ex

```diff
@@ -0,0 +1,432 @@
+
+    # produces %{parent_station_id => [%Alert{}, ...]}
+  end
+
+  defp get_parent_station_id_from_informed_entities(entities) do
+    entities
+    |> Enum.find_value(fn
+      %{stop: "place-" <> _ = parent_station_id} -> parent_station_id
+      _ -> false
+    end)
+  end
+
+  # defp parse_facility_data(facilities) do
+  #   facilities
+  #   |> Enum.map(fn %{"attributes" => %{"long_name" => long_name}, "id" => id} ->
+  #     %{name: long_name, id: id}
+  #   end)
+  # end
+
+  defp serialize_detail_page(
+         %Alert{header: header, informed_entities: entities} = alert,
+         %__MODULE__{
+           station_id_to_icons: station_id_to_icons,
+           facilities: facilities,
+           now: now
+         } = t
+       ) do
+    facility = get_facility_by_id(entities, facilities)
+
+    %DetailPage{
+      header_text: header,
+      icons: Map.fetch!(station_id_to_icons, parent_station_id(t)),
+      elevator_closure: serialize_closure(alert, facility, now)
+    }
+  end
+
+  defp serialize_list_page(
+         alerts,
+         %__MODULE__{} = t
+       ) do
+    stations =
+      alerts
+      |> alerts_by_station()
+      |> Enum.map(&serialize_station(&1, t))
+
+    %ListPage{
+      stations: stations
+    }
+  end
+
+  def priority(_instance), do: [2]
+
+  @spec serialize(t()) :: %{pages: list(DetailPage.t() | ListPage.t())}
+  def serialize(%__MODULE__{} = t) do
+    # case Screens.V3Api.get_json("alerts", %{
+    #        "filter[activity]" => "USING_WHEELCHAIR",
+    #        "include" => "facilities"
+    #      }) do
+    #   {:ok, result} ->
+    #     {:ok, stop_sequences} =
+    #       RoutePattern.fetch_stop_sequences_with_parent_stations("place-sull")
+
+    #     facilities =
+    #       get_in(result, [
+    #         "included",
+    #         Access.filter(&(&1["type"] == "facility"))
+    #       ])
+    #       |> parse_facility_data()
+
+    #     alerts = Screens.Alerts.Parser.parse_result(result)
+    #     parent_station_id = "place-sull"
+    #     now = DateTime.utc_now()
+    #   _ ->
+    #     IO.inspect(:error)
+    # end
+
+    active_at_home =
+      t
+      |> get_active_at_home_station()
+      |> Enum.map(&serialize_detail_page(&1, t))
+
+    active_elsewhere =
+      t
+      |> get_active_elsewhere()
+      |> serialize_list_page(t)
+
+    upcoming_at_home =
+      t
+      |> get_upcoming_at_home_station()
+      |> Enum.map(&serialize_detail_page(&1, t))
+
+    upcoming_on_connecting_lines =
+      t
+      |> get_upcoming_on_connecting_lines()
+      |> Enum.map(&serialize_detail_page(&1, t))
+
+    # first show detail pages for each closure at this station
+    # then if there is still space, show list pages for _all_ active closures across the system
+    # then if there is still space, show detail pages for upcoming closures at this station
+    # then if there is still space, show detail pages for active closures along lines serving this station
```

Comment:
Couple questions I have for this section: 

Where did the points for 385-88 come from?  Is it coming from the "Content Logic" page of the mocks?

Line 388 says "show detail pages for active closures along lines serving this station" (which matches the "Content Logic" page).  But on line 395, we're passing in `upcoming_on_connecting_lines` instead of an `active_on_connecting_lines` value.  I'm not sure the design ever uses an "upcoming on connecting lines" value

Reply:
Good catch. I'm not really sure what I was thinking here, but luckily the fix is easy. Just pushed up the changes.